--- Net_LDAP-0.6.6/usr/share/pear/Net/LDAP/Schema.php	2006-09-04 17:27:15.592199777 +0300
+++ Net_LDAP-0.6.6.bug-8286/usr/share/pear/Net/LDAP/Schema.php	2006-09-04 17:28:44.294183751 +0300
@@ -110,7 +110,8 @@
                      'syntaxes'          => &$this->_ldapSyntaxes );
 
         $key = strtolower($type);
-        return ((key_exists($key, $map)) ? $map[$key] : $this->raiseError("Unknown type $type"));
+        $ret = ((key_exists($key, $map)) ? $map[$key] : $this->raiseError("Unknown type $type"));
+        return $ret;
     }
     
     /**
--- Net_LDAP-0.6.6/usr/share/pear/Net/LDAP.php	2006-09-04 17:27:15.622200448 +0300
+++ Net_LDAP-0.6.6.bug-8286/usr/share/pear/Net/LDAP.php	2006-09-04 17:28:10.833435326 +0300
@@ -890,7 +890,8 @@
         $entry = $result->shift_entry();
         if (false === $entry) return $this->raiseError('Could not fetch RootDSE entry');
 
-        return new Net_LDAP_RootDSE($entry);
+        $ret = new Net_LDAP_RootDSE($entry);
+        return $ret;
     }
     
     /**
