--- php-pear-Net_LDAP-0.6.6/Net_LDAP-0.6.6/LDAP.php~	2004-03-08 12:42:44.000000000 +0100
+++ php-pear-Net_LDAP-0.6.6/Net_LDAP-0.6.6/LDAP.php	2004-12-05 01:11:50.943301208 +0100
@@ -26,8 +26,8 @@
 // $Id$
 
 require_once('PEAR.php');
-require_once('LDAP/Entry.php');
-require_once('LDAP/Search.php');
+require_once('Net/LDAP/Entry.php');
+require_once('Net/LDAP/Search.php');
 
 /**
  * Net_LDAP - manipulate LDAP servers the right way!
